composer.json

```diff
@@ -88,7 +88,7 @@
     "phpunit/phpunit": "~4.5",
     "phake/phake": "@stable",
     "friendsofphp/php-cs-fixer": "^1.10",
-    "phpdocumentor/phpdocumentor": "2.*"
+    "sensio/generator-bundle": "~2.3"
```

Comment:
generator bundle should be only installed in development mode (i.e: by the core team).